onnxruntime/hosting/server_configuration.h

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef ONNXRUNTIME_HOSTING_SERVER_CONFIGURATION_H
+#define ONNXRUNTIME_HOSTING_SERVER_CONFIGURATION_H
+
+#include <thread>
+#include <fstream>
+
+#include "boost/program_options.hpp"
+
+namespace onnxruntime {
+namespace hosting {
+
+namespace po = boost::program_options;
+
+// Wrapper around Boost program_options and should provide all the functionality for options parsing
+// Provides sane default values
+class ServerConfiguration {
+ public:
+  ServerConfiguration() {
```

Comment:
In the help message, we better describe what will be the default value.

Reply:
It is there

Reply:
Which is part of program_options